models/DataObject/ClassDefinition/CustomLayout.php

```diff
@@ -522,15 +522,15 @@ public function getDescription()
     }
 
     /**
-     * @param array $layoutDefinitions
+     * @param Layout $layoutDefinitions
```

Comment:
IMHO, this can be null as well (just tried to create a new class). Same for fielddefinitions. Getters&Setters.

Reply:
I added null to layoutDefinitions/fieldDefinitions